site/class-2-debug/index.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+/* Based on https://bl.ocks.org/mbostock/3887118 by Mike Bostock. */
+
+/* Define margins and size — https://bl.ocks.org/mbostock/3019563. */
+var
+  margin = {top: 48, right: 48, bottom: 48, left: 48},
+  width = 960 - margin.left - margin.right,
+  height = 500 - margin.top - margin.bottom
+
+/* Scales and axes. */
+var
+  x = d3.scaleLinear().range([0, width]),
+  y = d3.scaleLinear().range([height, 0]),
+  color = d3.scaleOrdinal().range(['#fe2f2f', '#feca2f', '#96fe2f']),
+  xAxis = d3.axisBottom(x),
+  yAxis = d3.axisLeft(y)
+
+/* Size SVG. */
+var svg = d3
+  .select('svg')
+  .attr('viewBox', [
+    0,
+    0,
+    margin.left + width + margin.right,
+    margin.top + height + margin.bottom
+  ].join(' '))
+  .append('g')
+  .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+
+/* Load data. */
+d3.csv('index.csv', row, onload);
+
+/* Handle data. */
+function onload(data) {
```

Comment:
Does this work?

Reply:
Yup, It does work. No clue why. I did fix the code, I overlooked the error there.